### DISCUSSION TITLE
Requesting functions in Feature Constructor for DateTime Datatype

### DISCUSSION THREAD

**Discussion by ganeshkamath89** [biolab/orange3#6505] [Ideas]
The Feature constructor has option to create a DateTime feature:
![image](https://github.com/biolab/orange3/assets/2648522/bcae420e-658e-4f9f-be87-974fd795a5f4)

However, if someone wants to extract Year, Month, Day, Hour, Minute, Second, Quarter, IsWeekend ... features related to Date and time, it seems very challenges to do it in Orange.

In the Feature Constructor, can these features be made available via select function dropdown, when user wants to create DateTime feature?

**Comment by simonaubertbd** [comment]
Hello @ganeshkamath89 Well, I totally upvote this ideas. The feature constructor is really too light as of now, especially compared by the huge number of function you can find in sql, excel, python, tableau, qlik, alteryx and so on. I guess it can be explained by the origine of Orange Data Mining.
Right now, it's my main issue after the lack of connectivity (like read and write from a database)

Best regards,

Simon
